lib/task/import/validate/csvDigitalObjectPathValidator.class.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * CSV Digital Object Path and URI check. Check digitalObjectPath and report:
+ *  - images not referenced from CSV
+ *  - images referenced in CSV but not found in image folder
+ *  - images referenced more that once in the CSV.
+ *
+ * @author     Steve Breker <sbreker@artefactual.com>
+ *
+ * @internal
+ * @coversNothing
+ */
+class CsvDigitalObjectPathValidator extends CsvBaseValidator
+{
+    const TITLE = 'Digital Object Path Test';
+
+    // Do not reset between CSVs.
+    protected $fileList = [];
+    protected $pathToDigitalObjects = '';
+
+    protected $digitalObjectPathColumnPresent;
+    protected $digitalObjectUriColumnPresent;
+    protected $digitalObjectUses = [];
+    protected $overriddenByUriCount = 0;
+
+    public function __construct(array $options = null)
```

Comment:
Nitpick: the type declaration for the `$options` argument should be explicitly set as nullable, e.g.

```
public function __construct(?array $options = null)
```

This will require updating the definition for `__construct()` for all of the subclasses of this class.